aya/src/sys/netlink.rs

```diff
@@ -377,7 +401,7 @@ impl NetlinkMessage {
                 ));
             }
             (
-                Vec::new(),
+                buf[data_offset + mem::size_of::<nlmsgerr>()..msg_len].to_vec(),
```

Comment:
is this right? shouldn't this be `[start...start+len]`?

Reply:
It's correct. Everything from the offset + sizeof(nlmsgerr ) to the end of the msg are attributes that we can parse later to get the error message.